data/error-manager.php

```diff
@@ -0,0 +1,45 @@
+<?php
```

Comment:
See `core/logger/manager.php`,  it should be implemented there as a logger: `core/logger/loggers/base.php`.

Reply:
Not relevant anymore.